src/mapping/config/MappingConfiguration.cpp

```diff
@@ -347,6 +350,10 @@ MappingConfiguration::ConfiguredMapping MappingConfiguration::createMapping(
       configuredMapping.mapping = PtrMapping(
           new RadialBasisFctMapping<VolumeSplines>(constraintValue, dimensions, VolumeSplines(), xDead, yDead, zDead));
     } else if (type == VALUE_RBF_GAUSSIAN) {
+      if (!std::isinf(supportRadius)) {
+        // Compute shape parameter from the support radius
+        shapeParameter = std::sqrt(-std::log(Gaussian::cutoffThreshold)) / supportRadius;
```

Comment:
Are you sure that there is not a "2" missing in front of the support radius?

Reply:
I copied the formula from the Gaussian class, so yes, pretty sure. Do you have a formula in mind?

Reply:
I looked at the documentation here: https://precice.org/tooling-rbf-shape.html
There it is also a bit ambiguous. What does "cover by the support radius" mean? A radius cannot cover sth, only the support :grin:
If you are sure, I am sure.